src/core/qgis.h

```diff
@@ -1386,6 +1386,20 @@ class CORE_EXPORT Qgis
     };
     Q_ENUM( DashPatternSizeAdjustment )
 
+    /**
+     * DpiMode enum
+     * \since QGIS 3.26
+     */
+    enum class DpiMode
+    {
+      All = 7, //!< All
+      Off = 0, //!< Off
+      QGIS = 1, //!< QGIS
+      UMN = 2, //!< UMN
```

Comment:
Just curious -- what actually is "UMN"? :laughing:

Reply:
I have no idea!

Reply:
Probably "University Of Minnesota", where mapserver comes from :)